pallets/capacity/src/lib.rs

```diff
@@ -151,6 +157,37 @@ pub mod pallet {
 		/// How much FRQCY one unit of Capacity costs
 		#[pallet::constant]
 		type CapacityPerToken: Get<Perbill>;
+
+		/// The number of blocks in a RewardEra
+		#[pallet::constant]
+		type EraLength: Get<u32>;
```

Comment:
I think we should use `Get<BlockNumberFor<Self>>` to ensures that your EraLength is compatible with the block number type of the runtime.

Reply:
I miss read that.